docs/src/pages/guides/examples.md

```diff
@@ -51,7 +51,7 @@ Using the above configuration, Monika will check the landing page every 10 secon
 
 ## HTML Form Submission Example
 
-Here is probe example with POST request to simulate HTML form submission
+Monika support POST methods. Belows is an example probe with a POST request to simulate HTML form submission:
```

Comment:
`You can also send POST requests using Monika. The following is an example of sending a POST request to simulate HTML form submission.`